modules/apps/user-associated-data/user-associated-data-api/src/main/java/com/liferay/user/associated/data/display/UADDisplay.java

```diff
@@ -53,16 +53,49 @@ public default String getEditURL(
 
 	public List<T> getRange(long userId, int start, int end);
 
+	/**
+	 * Returns an array of strings defining the field names to be used as table
+	 * column headers when sorting a list of entities of type {@code T}
+	 *
+	 * @return an array of field names used for column headers for sorting
+	 * @reivew
+	 */
 	public String[] getSortingFieldNames();
 
 	public String getTypeName(Locale locale);
 
+	/**
+	 * Returns an ordered range of all the entities of type {@code T} that match
+	 * the keywords with the given userId and groupIds.
+	 *
+	 * @param userId the userId whose data to search
+	 * @param groupIds the primary keys of the groups that the entities are
+	 *                 associated with
+	 * @param keywords the keywords which may occur in the entity's fields
+	 * @param orderByField the field to sort the entities by
+	 * @param orderByType the direction to sort the entities by, ascending or
+	 *                    descending
+	 * @param start the starting index of the result set
+	 * @param end the ending index of the result set
+	 * @return the ordered range of matching entities with the userId and
+	 * gorupIds
+	 * @review
+	 */
 	public List<T> search(
 		long userId, long[] groupIds, String keywords, String orderByField,
 		String orderByType, int start, int end);
 
-	public long searchCount(
-		long userId, long[] groupIds, String keywords, String orderByField,
-		String orderByType);
+	/**
+	 * Returns a count of the number of entities of type {@code T} that match
+	 * the keywords with the given userId and groupIds.
+	 *
+	 * @param userId the userId whose data to search
+	 * @param groupIds the primary keys of the groups that the entities are
```

Comment:
I just noticed this in the tips and realized that I did this incorrectly in my other pull.  When referring to a primary key of an entity type, we should actually say "primary key".  Looks like you did this right for "groupIds", but "userId" should say "the primary key of the user".

Reply:
See tip number 10 :)